### ISSUE TITLE
Unable to import ietf-ssh-server YANG module

### ISSUE THREAD

**Issue by umbertino** [sysrepo/sysrepo#2164]
Hi,

I'm trying to set up a demonstrator with sysrepo. When trying to import and install the **ietf-ssh-server** module this fails with the following error:
```
[INF]: Applying scheduled changes.
[WRN]: Missing required element "port" in "listening-ssh-server". (path: /ietf-ssh-server:listening-ssh-server)
[WRN]: Failed to apply some changes, leaving all changes scheduled.
[ERR]: Module "ietf-ssh-server" already scheduled for installation.
```
I do not really understand this message since the **ietf-ssh-server** YANG-module provides the missing port-element.
The module originates from [NETCONF-Central](http://netconfcentral.org/modulereport/ietf-ssh-server). Validating the module with pyang does not reveal any issue. Hence I assume it sane.

My current sysrepo module list is:

```
Module Name                | Revision   | Flags | Owner | Permissions | Submodules | Features                                                           
--------------------------------------------------------------------------------------------------------------------------------------------------------
ietf-datastores            | 2018-02-14 | I     | pi:pi | 666         |            |                                                                    
ietf-inet-types            | 2013-07-15 | i     |       |             |            |                                                                    
ietf-keychain              | 2015-10-09 | I     | pi:pi | 666         |            |                                                                    
ietf-netconf               | 2011-06-01 | I     | pi:pi | 666         |            | writable-running candidate rollback-on-error validate startup xpath
ietf-netconf-acm           | 2018-02-14 | I     | pi:pi | 666         |            |                                                                    
ietf-netconf-monitoring    | 2010-10-04 | I     | pi:pi | 666         |            |                                                                    
ietf-netconf-notifications | 2012-02-06 | I     | pi:pi | 666         |            |                                                                    
ietf-netconf-with-defaults | 2011-06-01 | I     | pi:pi | 666         |            |                                                                    
ietf-origin                | 2018-02-14 | I     | pi:pi | 666         |            |                                                                    
ietf-ssh-server            |            |  N    |       |             |            |                                                                    
ietf-x509-cert-to-name     | 2014-12-10 | I     | pi:pi | 666         |            |                                                                    
ietf-yang-library          | 2019-01-04 | I     | pi:pi | 666         |            |                                                                    
ietf-yang-metadata         | 2016-08-05 | i     |       |             |            |                                                                    
ietf-yang-types            | 2013-07-15 | i     |       |             |            |                                                                    
nc-notifications           | 2008-07-14 | I     | pi:pi | 666         |            |                                                                    
notifications              | 2008-07-14 | I     | pi:pi | 666         |            |                                                                    
sysrepo-monitoring         | 2020-04-17 | I     | pi:pi | 666         |            |                                                                    
yang                       | 2017-02-20 | I     | pi:pi | 666         |            |                                                                    


```

Do you have an idea?
My sysrepo-installation is
`sysrepoctl - sysrepo YANG schema manipulation tool, compiled with libsysrepo v1.4.70 (SO v5.5.12)`
running on a Raspberry Pi 4B.

Best Regards,

Umberto

**Comment by michalvasko** [comment]
Hi,
similar problem as #1995.

**Comment by umbertino** [comment]
Hi Michal,

thanks for this hint.
Same issue arises with "min-elements". Needed to comment this, too to get it running.

Cheers,

Umberto

**Comment by umbertino** [comment]
Solved according to [#1995](https://github.com/sysrepo/sysrepo/issues/1995).
